playgrounds/app/src/components/Editor.tsx

```diff
@@ -260,7 +260,7 @@ export default function Editor(props: EditorProps) {
         format: 'blob',
         width: canvasFrames[0].width,
         height: canvasFrames[0].height,
-        frames: canvasFrames,
+        frames: canvasFrames.map(el => ({ data: el.data.buffer, delay: (animationSeconds * 1000) / animationFPS}))
```

Comment:
Oh, cool, I will try this out, but if this is the main thing to solve this, I feel silly for missing it

Reply:
Yea, apparently you need to set a delay between frames when generating gifs.